docs/devices.md

```diff
@@ -55,79 +17,108 @@ For mice, the DPI column specifies if the mouse's sensitivity is fixed (`-`),
-| M215             | 1.0   | yes     |       |                                 |
-| M235             | 4.5   | yes     |       |                                 |
-| M305             | 1.0   | yes     |       |                                 |
-| M310             | 1.0   | yes     |       |                                 |
-| M315             |       | yes     |       |                                 |
-| M330             | ?.?   | yes     | ?     | smooth scrolling                |
-| MX 1100          | 1.0   | yes     | -     | smooth scrolling, side scrolling|
+You are able to read this feature using command-line interface of Solaar, but it is not possible to assign different keys.
+
+
+### Receivers:
+
+| USB ID    | Kind       | Max Paired Devices |
+------------|------------|--------------------|
+| 046d:c52b | Unifying   | 6                  |
+| 046d:c52f | Nano       | 2(?)               |
+| 046d:c532 | Unifying   | 6                  |
+| 064d:c534 | Nano(?)    | 2                  |
+| 064d:c539 | Lightspeed | 6                  |
+| 064d:c53a | Lightspeed | 6                  |
+| 064d:c53f | Lightspeed | 6                  |
+
+
+### Keyboards (Unifying):
+
+| Device           | WPID | HID++ | Battery | Other supported features                |
+|------------------|------|-------|---------|-----------------------------------------|
+| K230             | 400D | 2.0   | yes     |                                         |
+| K270             | 4003 | 2.0   | yes     |                                         |
+| K340             | 2007 | 1.0   | yes     |                                         |
+| K350             | 200A | 1.0   | yes     |                                         |
+| K360             | 4004 | 2.0   | yes     | FN swap, reprog keys                    |
+| K375s            | 4071 |       |         | FN swap                                 |
+| K400 Touch       | 400E | 2.0   | yes     | FN swap                                 |
+| K400 Touch       | 4024 | 2.0   | yes     | FN swap                                 |
+| K400 Plus        | 404D | 2.0   |         | FN swap                                 |
+| K520 		   | 2011 | 1.0   | yes     | FN swap                                 |
+| K750 Solar       | 4002 | 2.0   | yes     | FN swap, Lux reading, light button      |
+| K780             | 405B | 4.5   | yes     | FN swap                                 |
+| K800 Illuminated | 2010 | 1.0   | yes     | FN swap, reprog keys, LEDs              |
+| K800 (new ver)   | 406E | 4.5   | yes     | FN swap                                 |
+| K830 Illuminated | 4032 | 2.0   | yes     | FN swap                                 |
+| N545             | 2006 |       | yes     | 	 				      |
+| TK820            |      | 2.0   | yes     | FN swap                                 |
+| Craft            | 4066 | 4.5   |         | 	       	      			      |
```

Comment:
While you're at it mind fixing the tabs used in these tables? They should be replaced with spaces :grin: